xm-commons-domainevent/src/main/java/com/icthh/xm/commons/domainevent/config/DomainEventConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.icthh.xm.commons.domainevent.config;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@RequiredArgsConstructor
+@ConditionalOnProperty("application.timeline-domain-event-enabled")
```

Comment:
I think property should not contain `timeline` mention (application.timeline-domain-event-enabled)
because domain events are broader then just timeline.